packages/web-components/src/components/va-button-pair/va-button-pair.scss

```diff
@@ -73,6 +75,6 @@
   }
 
   :host([continue]:not([continue='false']):not([uswds])) va-button::part(button) {
-    width: 130px;
+    width: 280px;
```

Comment:
I made this change to be consistent with the other buttons. The continue was the only one that shrank to 130px below a 320px screen size. I was trying to find a design to see if that was intentional, but was unable to. I can reverse this if I need to.

Reply:
Sounds good. Just double checking because without the Storybook link working, I couldn't quickly check how it was displaying compared to the other stacking v1 buttons. I think using the working ones as a reference makes sense.

Reply:
I can't think of a use case for a button to shrink below 320px. In fact, we shouldn't be doing anything differently across the site from 0px up until 479px. There are some odd rules in our CSS where the breakpoint doesn't kick in until 320px which I don't understand.